postgres/values.yaml

```diff
@@ -13,5 +13,6 @@ password: DEFINED-IN-REGION-CHART
 port_public: '5432'
 port_private: '5432'
 port_metrics: '9113'
+hba_cidr: 100.0.0.0/8
```

Comment:
Its the most pragmatic at the moment - otherwise we have to overwrite values for each chart with a DB in the region chart